sys/mips/mips/pm_machdep.c

```diff
@@ -310,13 +310,17 @@ sendsig(sig_t catcher, ksiginfo_t *ksi, sigset_t *mask)
 	regs->pc = (trapf_pc_t)catcher;
 	regs->c12 = regs->pc;
 	regs->csp = sfp;
-	regs->c17 = td->td_pcb->pcb_cherisignal.csig_sigcode;
+	regs->c17 = p->p_md.md_sigcode;
 
 	/*
 	 * For now only change IDC if we were sandboxed. This makes cap-table
 	 * binaries work as expected (since they need cgp to remain the same).
 	 *
 	 * TODO: remove csigp->csig_idc
+	 *
+	 * XXX: DDC used to be unconditional but was made conditional
+	 * as part of a commit to support legacy ABI.  Should it be
+	 * conditional on CHERIABI_LEGACY_SUPPORT?
```

Comment:
CHERIABI_LEGACY_SUPPORT got ripped out, it was the old pre-captable ABI (or maybe an intermediate version too).